blog.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+    <link href="https://fonts.googleapis.com/css?family=Merriweather:300,700|Public+Sans:300,700&display=swap" rel="stylesheet">
+    <title>Adopt-A-Dog: Blog</title>
+  </head>
+
+  <body>
+    <header>
+      <div class="header">
+          <div class="header-logo">
+            <a href="index.html"><img src="images/adoptadog-logo.svg" alt="Adopt-a-dog logo"></a>
+            <h1>Seattle's premiere dog adoption agency</h1>
+          </div>
+          <div class="header-checkout">
+            <p><img src="images/cart-icon.svg" width="30" alt="shopping cart">
+            $1234.56</p>
+            <a href="checkout.html">Checkout</a>
+          </div>
+      </div>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li>/</li>
+          <li><a href="dogs.html">Dogs</a></li>
+          <li>/</li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="container">
+      <article>
+        <h2 id="blog-h1">Title Goes Here</h2>
+          <div class="blog-article">
+            <section id="blog-1">
```

Comment:
Ideally you would not even have empty HTML skeleton elements to target. You should be able to look in the 
<div class="container">
      <article>
And create and insert new elements based on how much information is in your database or array/ object.